AdditionalQuestsCode/AdditionalQuestsCode/AdditionalQuestsCode.csproj

```diff
@@ -1,88 +1,70 @@
+    <Reference Include="$(GameFolder)\Modules\SandBoxCore\bin\Win64_Shipping_Client\*.dll">
+      <HintPath>%(Identity)</HintPath>
       <Private>False</Private>
     </Reference>
-    <Reference Include="TaleWorlds.ObjectSystem">
-      <HintPath>D:\SteamLibrary\steamapps\common\Mount &amp; Blade II Bannerlord\bin\Win64_Shipping_Client\TaleWorlds.ObjectSystem.dll</HintPath>
-      <Private>False</Private>
-    </Reference>
-    <Reference Include="TaleWorlds.SaveSystem">
-      <HintPath>D:\SteamLibrary\steamapps\common\Mount &amp; Blade II Bannerlord\bin\Win64_Shipping_Client\TaleWorlds.SaveSystem.dll</HintPath>
+    <Reference Include="$(GameFolder)\Modules\StoryMode\bin\Win64_Shipping_Client\*.dll">
+      <HintPath>%(Identity)</HintPath>
       <Private>False</Private>
     </Reference>
+	<Reference Include="System.Windows.Forms" />
   </ItemGroup>
-  <ItemGroup>
-    <Compile Include="AdditionalQuestsSubModule.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Quests\AdditionalQuestsHelperMethods.cs" />
-    <Compile Include="Quests\HeadmanNeedsMilitiaWeaponsIssueBehavior.cs" />
-    <Compile Include="Quests\TownUprisingIssueBehavior.cs" />
-    <Compile Include="Quests\StarvingTownNeedsFoodIssueBehavior.cs" />
-    <Compile Include="Quests\NobleWantsTrainingBattleIssueBehavior.cs" />
-    <Compile Include="Quests\VillageBanditArmyRaidIssueBehavior.cs" />
+
+  <ItemGroup>    
+    <PackageReference Include="Bannerlord.BUTR.Shared" Version="1.6.1.39">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="Nullable" Version="1.3.0">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="IsExternalInit" Version="1.0.0">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="Bannerlord.BuildResources" Version="1.0.0.33">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="Bannerlord.ButterLib" Version="1.0.21" IncludeAssets="compile" />
+    <PackageReference Include="Bannerlord.UIExtenderEx" Version="2.1.1" IncludeAssets="compile" />
+    <PackageReference Include="Bannerlord.MCM" Version="4.3.4" IncludeAssets="compile" />
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>compile</IncludeAssets>
+    </PackageReference>
```

Comment:
What downloads do we need to compile?
the URL links would be helpful, pop it in the readme under a "Dependencies" header

Reply:
If you go look at what is bundled already with the vanilla game, Newtonsoft.Json is one of them. We do not need to handle bundling this as it's already provided.

Reply:
ah, I meant the other libraries
These ones:
Bannerlord.MCM - https://www.nexusmods.com/mountandblade2bannerlord/mods/612
Bannerlord.UIExtenderEx - https://www.nexusmods.com/mountandblade2bannerlord/mods/2102
Bannerlord.ButterLib - https://www.nexusmods.com/mountandblade2bannerlord/mods/2018
Bannerlord.BUTR.Shared - https://www.nexusmods.com/mountandblade2bannerlord/mods/2513
Just double checking that we have the same ones for these

Reply:
Ah, nvm I see that nuget handles all those packages, nice